package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "scorer-ui-kit",
-  "version": "1.7.9",
+  "version": "1.8.0-beta.7",
```

Comment:
Don't merge agains main if you are doing for beta, please 🙏

Reply:
fixed